lib/store/settings_store.dart

```diff
@@ -162,6 +169,7 @@ abstract class SettingsStoreBase with Store {
 
   static const defaultPinLength = 4;
   static const defaultActionsMode = 11;
+  static const defaultPinCodeTimeOutDuration = 10;
```

Comment:
I would argue to make the default value as `PinCodeRequiredDuration defaultPinCodeTimeOutDuration = PinCodeRequiredDuration.tenminutes;` to avoid any error by mistakenly changing this integer